scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -535,6 +535,7 @@ def _update_instance_info(self, harness_state, handler_time, flash_error):
             self.instance.status = harness_state
             if harness_state == "failed":
                 self.instance.reason = "Failed"
+            self.instance.add_missing_case_status("blocked", "Failed")
```

Comment:
should the missing cases have reason set to the harness state instead of always 'Failed' ?
```suggestion
            self.instance.reason = "Harness state: " + harness_state
            self.instance.add_missing_case_status("blocked")
```